liwords-ui/src/chat/chat.tsx

```diff
@@ -373,6 +386,33 @@ export const Chat = (props: Props) => {
     userID,
   ]);
 
+  useEffect(() => {
+    if (selectedChatTab === 'PLAYERS') {
+      const lastMessage = chatEntities.reduce(
+        (acc: ChatEntityObj | undefined, ch) =>
+          'timestamp' in ch &&
+          (acc === undefined || ch.timestamp! > acc.timestamp!)
+            ? ch
+            : acc,
+        undefined
+      );
+      if (
+        lastMessage?.channel !== 'chat.lobby' &&
+        (lastMessage?.timestamp || 0) > lastNotificationTimestamp
+      ) {
+        setNotificationCount((x) => x + 1);
+        setLastNotificationTimestamp((x) =>
+          lastMessage?.timestamp || 0 > x ? lastMessage?.timestamp || x : x
```

Comment:
if [without parentheses](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#table) the ternary's condition is `lastMessage?.timestamp || (0 > x)`, it doesn't look right?

sidenotes:
- `lastMessage?.timestamp` (from golang) and `lastNotificationTimestamp` (from browser's `Date.now()`) may not be in sync -- just a note, maybe not trivial to fix
- `??` conveys the intent slightly better (but it's ok to use `||`)

Reply:
Ignoring the sidenotes for now. We should address the time issue if we see problems.